lib/lightning/invocation.ex

```diff
@@ -385,6 +385,16 @@ defmodule Lightning.Invocation do
     )
   end
 
+  def search_workorders_having_dataclips_query(
+        %Project{} = project,
+        %SearchParams{} = params
+      ) do
+    project
+    |> search_workorders_query(params)
+    |> join(:inner, [workorder: wo], assoc(wo, :dataclip), as: :dataclip)
+    |> where([dataclip: d], is_nil(d.wiped_at))
+  end
```

Comment:
It looks like the two places that call this function were also the only two places that called `search_workorders` - why not extend the existing function? Where would we not want to have this behaviour?

Reply:
There's a 3rd place where `search_workorders` is being used, loading workorders for the whole history page.

Reply:
Ah all right, perhaps rearranging this to be like:

```
project
|> search_workorders_query(params)
|> exclude_wiped_dataclips()
```

How does that sound?

Reply:
Hmm, yeah, that looks better. Let me update it